src/core/org/apache/jmeter/gui/MainFrame.java

```diff
@@ -680,7 +683,15 @@ private void addQuickComponentHotkeys(JTree treevar) {
 
             @Override
             public void actionPerformed(ActionEvent actionEvent) {
-                String propname = "gui.quick_" + actionEvent.getActionCommand();
+                //Bug 62336
+                AWTEvent current_event = EventQueue.getCurrentEvent();
+                String key_text = "";
+                if(current_event instanceof KeyEvent) {
```

Comment:
I would place a space between `if` and the opening parenthesis as `if` is not a function call.